netlify.toml

```diff
@@ -0,0 +1,13 @@
+[build]
+  command = """
+    rustup install nightly --profile minimal && cargo doc --no-deps
+    """
+  publish = "target/doc"
+
+[build.environment]
+  RUSTDOCFLAGS="""
+    -D warnings \
```

Comment:
I think it makes sense to drop `-D warnings` here. It can still be useful to view documentation even if there are warnings, and we have another CI check that will catch the warning.

Reply:
I don't feel too strongly about this; happy to do whatever.

Reply:
yeah, we have a separate check that docs build without warnings: https://github.com/tokio-rs/tokio/blob/0b8a3c34a1a119a61801b4e7e287f91684d11941/.github/workflows/ci.yml#L370-L385

so, we should always catch failing docs builds on CI. i agree with Alice, let's always build the netlify preview even if something is broken
```suggestion
```

Reply:
do we want to make that check as part of this PR or a different PR? happy with whichever; more keen on reducing maintenance overhead.

Reply:
The CI check for building documentation already exists.